draft-ietf-tls-tls13.md

```diff
@@ -1914,19 +1916,23 @@ Structure of this message:
 %%% Key Exchange Messages
 
        struct {
-           ProtocolVersion version;
+           ProtocolVersion legacy_version = 0x0303;    /* TLS v1.2 */
            Random random;
+           opaque legacy_session_id_echo<0..32>;
            CipherSuite cipher_suite;
+           uint8 legacy_compression_method = 0;
            Extension extensions<6..2^16-1>;
        } ServerHello;
 
 version
-: This field contains the version of TLS negotiated for this connection.  Servers
-  MUST select a version from the list in ClientHello's supported_versions extension,
-  or otherwise negotiate TLS 1.2 or a previous version.
-  A client that receives a version that was not offered MUST abort the handshake.
-  For this version of the specification, the version is 0x0304.  (See
-  {{backward-compatibility}} for details about backward compatibility.)
+: In previous versions of TLS, this field was used for version negotiation
+  and represented the selected version number for the connection. Unfortunately,
+  some middleboxes fail when presented with new value. 
+  In TLS 1.3, the TLS server indicates its version using the
+  "supported_versions" extension ({{supported-versions}}),
+  and the legacy_version field MUST
+  be set to 0x0303, which is the version number for TLS 1.2.
```

Comment:
Do we want to specify a client alert to send when receiving some other value?

Reply:
+1. In case `legacy_version` is not set to 0x0303 and `supported_versions` is present, the connection should fail with `illegal_parameter`

though it probably should be placed in the `legacy_version` description to mirror how Client Hello is described

Reply:
Actually, we should ignore it to match how we handle ClientHello.legacy_version

Reply:
And that's what the text already says.